pages/landing.js

```diff
@@ -1,31 +1,53 @@
 import React from 'react';
-import firebase from 'firebase';
-import Card, {
-  CardHeader,
-  CardTitle,
-  CardContent,
-  CardButtonContainer,
-} from '../components/card';
-import { EDIT } from '../constants';
-import Button from '../components/button';
+import styled from 'styled-components';
+import Sidebar from '../components/sidebar';
+import { getHackathons } from '../utility/firebase';
+import nwPlusLogo from '../assets/nwplus.svg';
+import { useAuth } from '../utility/auth';
 
-export default function Landing() {
-  return (
-    <>
-      <div>Login Page</div>
+const Container = styled.div`
+  display: flex;
+  align-items: stretch;
+  min-height: 100vh;
+`;
+
+const NwPlusImage = styled.img`
+  margin-top: 5%;
+  width: 250px;
+  height: 250px;
+  margin-bottom: 2%;
+`;
+
+const WelcomeTitle = styled.h1`
+  font-size: 48px;
+`;
 
-      <Card>
-        <CardHeader>
-          <CardTitle>Hi</CardTitle>
-          <p>Some extra text</p>
-          <CardButtonContainer>
-            <Button type={EDIT}>Hi there</Button>
+const InfoMessage = styled.p`
+  font-size: 32px;
+`;
 
-            <Button onClick={() => firebase.auth().signOut()}>Sign out</Button>
-          </CardButtonContainer>
-        </CardHeader>
-        <CardContent>Example usage of card component</CardContent>
-      </Card>
-    </>
+export default function Landing({ hackathons, id }) {
+  const { user } = useAuth();
+  return (
+    <Container>
+      <Sidebar currentPath={id} hackathons={hackathons} />
+      <div style={{ width: '80vw' }}>
+        <div style={{ width: '100%', textAlign: 'center' }}>
+          <NwPlusImage src={nwPlusLogo} />
+          <WelcomeTitle>Welcome to the CMS {user.displayName}!</WelcomeTitle>
+          <InfoMessage>Please choose a hackathon from the sidebar.</InfoMessage>
+        </div>
+      </div>
+    </Container>
   );
 }
+
+export const getStaticProps = async ({ params }) => {
+  const hackathons = await getHackathons();
+  return {
+    props: {
+      hackathons,
+      id: params?.id ?? null,
```

Comment:
Do we need id on landing page?

Reply:
No, not sure why I kept it, I'll remove it.

Reply:
Fixed!